api-reference/upload-file-api/README.md

```diff
@@ -37,14 +37,27 @@ ImageKit.io allows you to upload a file with the following mime types:
 | webp               | WebP image files       |
 | gif                | GIF image files        |
 | svg                | SVG image files        |
+| heif               | HEIF image files       |
+| heic               | HEIC image files       |
+| bmp                | BMP image files        |
+| flif               | FLIF image files       |
+| arw                | ARW image files        |
+| tif                | TIF files              |
+| tiff               | TIFF files             |
 | pdf                | PDF documents          |
 | js                 | Javascript files       |
+| mjs                | MJS javascript files   |
 | css                | CSS files              |
+| php                | PHP files              |
+| py                 | PY files               |
+| yml                | YML files              |
+| yaml               | YAML files             |
 | woff2              | Font files             |
 | woff               | Font files             |
 | ttf                | Font files             |
 | otf                | Font files             |
 | eot                | Font files             |
+| ps                 | Font files             |
```

Comment:
yes

Reply:
It's a vector graphic file according to Adobe. It can contain both graphics and text:
https://www.adobe.com/creativecloud/file-types/image/vector/ps-file.html#ps